compiler/optimizer/LocalDeadStoreElimination.cpp

```diff
@@ -1151,8 +1148,7 @@ void TR::LocalDeadStoreElimination::killStoreNodes(TR::Node *node)
          TR::SymbolReference *storeSymRef=storeNode->getSymbolReference();
 
          // TODO: improve by not killing stores that are definitely disjoint
-         // if (node->getSymbolReference()->getUseDefAliases().contains(storeSymRef, comp()))
-         if (usedef_vector[storeSymRef->getReferenceNumber()])
+         if (node->getSymbolReference()->getUseDefAliases().contains(storeSymRef, comp()))
```

Comment:
From the looks of it, it was added before we split and released OMR; it was done by another project where the change actually improved CPU time for them considerably.